samples/artifacts/Data Transformation/TimeExtensionSample.siddhi

```diff
@@ -0,0 +1,39 @@
+@App:name("TimeExtensionSample")
```

Comment:
Use some more functions in time extension. Also, find a sample which is relevant to the requirement.